WordPress/src/main/res/values/strings.xml

```diff
@@ -2247,7 +2248,9 @@
     <string name="wordpress_help_center_button">WordPress help center</string>
     <string name="browse_our_faq_button">Browse our FAQ</string>
     <string name="my_tickets">My Tickets</string>
```

Comment:
Same here, My Tickets should go

Reply:
Removed: e1f1d73a54abf855e7f4de45414f98916a64d4b5